From f076eb444565db967dbf493606eff3f0022d7b52 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Mon, 1 Oct 2018 13:04:10 +0530
Subject: [PATCH 0433/1852] dt-bindings: csi2rxss: Enable VCX support

commit 057fd3a4b5d364568d990ca2cedd2fbb9cdc419b from
https://github.com/Xilinx/linux-xlnx.git

A new optional dt bool property xlnx,en-vcx is now added to the dt node
of the MIPI CSI2 Rx Subsystem. When this is enabled, the maximum number
of virtual channels is 16 else 4.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt        | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
index 3d948eaad402..8325f7e607b0 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
@@ -59,6 +59,10 @@ Required properties:
 - data-lanes: The number of data lanes through which CSI2 Rx Subsystem is
   connected to the camera sensor as per video-interfaces.txt
 
+Optional Properties
+
+- xlnx,en-vcx: When present, the max number of virtual channels can be 16 else 4.
+
 Example:
 
 	csiss_1: csiss@a0020000 {
-- 
2.31.1

